fhirr4/native/src/main/java/io/ballerinax/fhirr4/Utils.java

```diff
@@ -31,7 +34,13 @@ public class Utils {
      * @return {@code true} if the length of the resource path array is equal to the size of the provided list of paths,
      *         {@code false} otherwise
      */
-    static boolean isPathsMatching(String[] resourcePaths, String[] requestPaths) {
+    static boolean isPathsMatching(String[] resourcePathsWithNoBasePath, String[] basePath, String[] requestPaths) {
```

Comment:
'resourcePaths' is the correct term for 'resourcePathsWithNoBasePath'. correct term for 'basePath' is 'servicePath'

Rename parameters to following:
resourcePathsWithNoBasePath -> resourcePaths
basePath -> servicePath